src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -50,10 +53,17 @@ const SSICredentialsRequiredScreen: FC<Props> = (props: Props): JSX.Element => {
           restrictToFormats: format,
           restrictToDIDMethods: subjectSyntaxTypesSupported,
         });
-        const matchedVCs: Array<UniqueVerifiableCredential> = selectResult.verifiableCredential
-          ? selectResult.verifiableCredential
-              .map((matchedVC: IVerifiableCredential) => getMatchingUniqueVerifiableCredential(uniqueVCs, matchedVC))
-              .filter((matchedVC): matchedVC is UniqueVerifiableCredential => !!matchedVC) // filter out the undefined (should not happen)
+        if (selectResult.areRequiredCredentialsPresent === "error") {
```

Comment:
should we not check against Status.ERROR?

Reply:
Ah it has an enum. So yes, done